docs/schema_plugins.md

```diff
@@ -1 +1,234 @@
 # Extending NOMAD-Simulations to support custom methods and outputs
+
+As you develop your parser, you may find that the `nomad-simulations` package does not include some relevant quantities for your particular use case. You can easily extend upon the schema by adding your own custom schema under the `schema_packages/` directory in your parser plugin. For this, you should utilize and build upon `nomad-simulations` for consistency and future compatibility or integration. The following page demonstrates the basics of how to implement your own schema in practice.
```

Comment:
> you should utilize and build upon `nomad-simulations` for consistency and future compatibility or integration

Exactly, I'd even say that _we provide a foundation that already manages most of the messy logic_.
The compatibility is more so on the semantical than code side, i.e. it'll still run as long as they use the core NOMAD classes.